Sources/Sentry/SentrySDK.m

```diff
@@ -53,6 +55,16 @@ + (void)setCrashedLastRunCalled:(BOOL)value
     crashedLastRunCalled = value;
 }
 
++ (SentryAppStartMeasurement *)appStartMeasurement
+{
+    return appStartMeasurement;
+}
+
++ (void)setAppStartMeasurement:(SentryAppStartMeasurement *)value
+{
+    appStartMeasurement = value;
+}
```

Comment:
M: Is this a public API? can we hide it? we're detecting it automatically, people should not touch it

Reply:
Nope, it's only for internal. I added a comment.